source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -145,7 +147,7 @@ private void spawnGhostKing() {
     GridPoint2 maxPos = terrain.getMapBounds(0).sub(2, 2);
 
     GridPoint2 randomPos = RandomUtils.random(minPos, maxPos);
-    Entity ghostKing = NPCFactory.createGhostKing(player);
+    Entity ghostKing = NPCFactory.createDog(player);
```

Comment:
Why is the ghost king a dog

Reply:
To keep the spawning mechanics the same as the base game since that's for the map/level team to implement.